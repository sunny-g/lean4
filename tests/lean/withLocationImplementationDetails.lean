axiom A : Nat
axiom B : Nat
axiom equality : A = B

example : A = 3 -> B = 3 := by
  intro h
  rewrite [equality] at *
