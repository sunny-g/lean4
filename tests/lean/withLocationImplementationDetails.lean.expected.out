withLocationImplementationDetails.lean:5:28-7:25: error: unsolved goals
h : B = 3
‚ä¢ B = 3
